packages/sdk-codegen/src/python.gen.ts

```diff
@@ -299,7 +341,11 @@ ${this.hooks.join('\n')}
     } else {
       propType = `Optional[${mappedType.name}] = ${this.nullStr}`
     }
-    return `${indent}${property.name}: ${propType}`
+    let propName = property.name
+    if (this.pythonKeywords.has(propName)) {
+      propName = propName + '_'
+    }
+    return `${indent}${propName}: ${propType}`
```

Comment:
again, `LookmlModelExploreJoins` getting a constructor exposed a bug here in the constructor signature